docker-compose.yml

```diff
@@ -41,19 +41,31 @@ services:
             - AXIS_CTXAVAILABLETRANSPORTS=http;tchannel
 
             - BEHAVIOR_RAW=client,server,transport
+            - BEHAVIOR_SKIP_RAW=client=java+transport=tchannel,server=java+transport=tchannel
```

Comment:
Yes, this is very very powerful now. We can select and pick what tests to run while still respecting the "full blown cross product of axis" behavior.